railties/lib/rails/generators/rails/app/templates/config/environments/test.rb.tt

```diff
@@ -30,6 +30,9 @@ Rails.application.configure do
   # Render exception templates for rescuable exceptions and raise for other exceptions.
   config.action_dispatch.show_exceptions = :rescuable
 
+  # Raise exceptions instead of rendering exception templates
+  # config.action_dispatch.show_exceptions = :none
+
```

Comment:
```suggestion
```

There is no reason for us to list what are other possible values. We don't do with any other config.

Reply:
I still think we should delete those lines.

Reply:
Sorry, I missed them. I've removed them.